todoapp/build.gradle

```diff
@@ -1,11 +1,33 @@
+// Define versions in a single place
+ext {
+    // Sdk and tools
+    minSdkVersion = 14
+    targetSdkVersion = 26
+    compileSdkVersion = 26
+    buildToolsVersion = '26.0.2'
+
+    // App dependencies
+    supportLibraryVersion = '26.1.0'
+    guavaVersion = '22.0-android'
+    junitVersion = '4.12'
+    mockitoVersion = '2.8.47'
+    hamcrestVersion = '1.3'
+    runnerVersion = '1.0.1'
+    rulesVersion = '1.0.1'
+    espressoVersion = '3.0.1'
+    roomVersion = "1.0.0"
+}
+
 buildscript {
-    ext.kotlin_version = '1.1.3-2'
+    ext.kotlinVersion = '1.1.51'
+    ext.gradleVersion = '3.0.0'
```

Comment:
to avoid misunderstandings, I would call this gradlePluginVersion (gradle is 4.1)

Reply:
Done.